COCOAPODS.md

```diff
@@ -0,0 +1,150 @@
+    cocoapods {
+        // Configure fields required by CocoaPods.
+        summary = "Some description for a Kotlin/Native module"
+        homepage = "Link to a Kotlin/Native module homepage"
+    }
+}
+```
+
+</div>
+
+> __Note:__ the following podspec fields are required by CocoaPods:
+> 
+>  - `version`
+>  - `summary`
+>  - `homepage`
+>
+> A version of the Gradle project is used as a value for the `version` field.
+Fields`summary` and `homepage` can be configured using the `cocoapods` code block.
+
+This podspec file can be referenced from a [Podfile](https://guides.cocoapods.org/using/the-podfile.html)
+of an Xcode project. After that the framework built from the Kotlin/Native module can be used from
+this Xcode project. If necessary, this framework is automatically rebuilt during Xcode build process.
+
+### Workflow
+
+The following steps allows one to import a Kotlin/Native module in an existing Xcode project.
+
+0. [Install](https://guides.cocoapods.org/using/getting-started.html#installation) CocoaPods.
+It's recommended to use CocoaPods 1.6.1 or higher.
+
+1. Configure a Gradle project: apply the `org.jetbrains.kotlin.native.cocoapods` plugin, add
+and configure the targets needed and specify the required podspec fields.
+
+2. Run the `podspec` task. The podspec file described above will be generated.
+
+    > In order to avoid compatibility issues during an Xcode build, the plugin requires using
+    a [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html). It's
+    possible to generate the wrapper automatically during execution of the `podspec` task
+    by running it with an additional parameter `-Pkotlin.native.cocoapods.generate.wrapper=true`.
+
+3. Add a reference to the generated podspec in a Podfile of the Xcode project.
+
+    <div class="sample" markdown="1" theme="idea" mode="ruby">
+
+    ```ruby
+    target 'my-ios-app' do
+        pod 'my_kotlin_library', :path => 'path/to/my-kotlin-library'
+    end
+    ```
+
+    </div>
+
+4. Run `pod install` for the Xcode project.
+    
+After these steps the Kotlin/Native module is included in the Xcode project. Now it's possible
+to open the generated workspace (see [CocoaPods documentation](https://guides.cocoapods.org/using/using-cocoapods.html#installation))
+and run an Xcode build.
+
+### Interoperability
+
+The CocoaPods plugin also allows using CocoaPods libraries without manual configuring cinterop
+parameters (see [a corresponding section](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#cinterop-support)
+in the multiplatform plugin documentation). The `cocoapods { ... }` code block allows one to
+add dependencies on CocoaPods libraries.
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin">
+
+```kotlin
+kotlin {
+    cocoapods {
+        // Configure a dependency on AFNetworking.
+        // The CocoaPods version notation is supported.
+        // The dependency will be added to all macOS and iOS targets.
```

Comment:
Can we control which target  dependency is added to?

Reply:
Currently no.